### PR TITLE
Support the option to add tags to newly created Detective graphs

### PR DESCRIPTION
**Pull request by alaniyonu**
Introduces an optional parameter that enables tagging newly created graphs as part of the account setup process.

Note: This PR cannot be merged until an SDK with the Tagging operations for Detective is released.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/test_scripts.py [hunk 1]
```diff
@@ -24,6 +24,14 @@ def test_setup_command_line_enableDetective():
 
     args = enableDetective.setup_command_line(['--master_account', '000000000001', '--assume_role', 'detectiveAdmin', '--enabled_regions', 'us-east-1,us-east-2,us-west-2,ap-northeast-1,eu-west-1', '--input_file', 'accounts.csv'])
     assert args.master_account == '000000000001'
+    assert args.tags == None
+
+    args = enableDetective.setup_command_line("--master_account 123456789012 --assume_role detectiveAdmin --input_file accounts.csv --tags TagKey1=TagValue1,TagKey2=TagValue2,TagKey3=TagValue3".split(" "))
+    assert args.tags == {
+        "TagKey1": "TagValue1",
+        "TagKey2": "TagValue2",
+        "TagKey3": "TagValue3",
```
Comment (wingamajig) [inline tests/test_scripts.py:33]: Can you test key with no value set?

Also, do we test negative cases here? Might be good to test a case of invalid tags (no key, key/value with invalid characters, 51+ tags)

Comment (alaniyonu) [reply tests/test_scripts.py:33]: This test is primarily focused on parsing the command line arguments for the script. To that end, I'll adding a case for a tag key with no associated value.

Adding the other test cases that you mentioned doesn't make as much sense to me, since those constrains are enforced by the service and also checked by the boto3 client.

To add those cases, we would either need to duplicate the implementation and enforcement of those constrains here, or effectively transform these tests into integration tests, to verify that the boto3 client and the Detective service are enforcing the constrains that they claim to posses. I do not see much value in either option.

Comment (wingamajig) [reply tests/test_scripts.py:33]: looks good!

**Review by wingamajig — APPROVED** [review]

**Review by wingamajig — CHANGES_REQUESTED** [review]
I just noticed that the README has the managed policy, and does not include tagging operations but probably should.

**Review by wingamajig — APPROVED** [review]
ahhh, sorry - you are calling createGraph with the arg so no need to update!

**Comment by wingamajig** [comment]
Abiola, one more thing - I think we should update the args in the README to include --tags?

**Comment by alaniyonu** [comment]
For documentation - the plan is to deprecate the README and point to the official documentation as the most up-to-date source of information for this script - https://docs.aws.amazon.com/detective/latest/adminguide/detective-github-scripts.html

We'll be updating the docs there to include information on how to tag Detective graphs created by this script early this week.
